xonsh/proc.py

```diff
@@ -31,6 +31,8 @@
 from xonsh.lazyasd import lazyobject, LazyObject
 from xonsh.jobs import wait_for_active_job
 from xonsh.lazyimps import fcntl, termios, _winapi, msvcrt, winutils
+# these decorators are imported for users back-compatible
+from xonsh.tools import unthreadable, foreground, uncapturable  # NOQA
```

Comment:
Yes we do. Thanks for doing that.  You can keep `foreground` in proc, though, since that was deprecated already.